core/pom.xml

```diff
@@ -63,6 +63,7 @@
     <java-websocket.version>1.6.0</java-websocket.version>
     <jackson.version>2.19.0</jackson.version>
     <okhttp.version>4.12.0</okhttp.version>
+    <langchain4j.version>1.0.1</langchain4j.version>
```

Comment:
This wouldn't go here... witht the `contrib/langchain4j` _"module"_ it (now) belongs in that POM, agreed?

Maybe it's easier to wait for https://github.com/google/adk-java/pull/155, first; up to you.

Reply:
@glaforge sorry not remove, of course, but move from here (`core`) into the `contrib/langchain/pom.xml` would be "cleaner", no?

Reply:
Should it be in core if it's specific to that module?